src/interpolations.jl

```diff
@@ -204,6 +204,39 @@ getnbasefunctions(::Interpolation)
 #   edgedof: dof on a line between 2 vertices (i.e. "corners") (3D only)
 #   celldof: dof that is local to the element
 
+"""
+    shape_values!(values::AbstractArray{T}, ip::Interpolation, ξ::Vec)
+
+Evaluate all shape functions of `ip` at once at the reference point `ξ` and store them in `values`.
+"""
+function shape_values!(values::AT, ip::IP, ξ::Vec) where {IP <: Interpolation, AT <: AbstractArray}
+    @inbounds for i in 1:getnbasefunctions(ip)
```

Comment:
Shouldn't there be a `checkbounds` here as default, which can be removed by calling `@inbounds shape_values!(...)`?

```suggestion
@propagate_inbounds function shape_values!(values::AT, ip::IP, ξ::Vec) where {IP <: Interpolation, AT <: AbstractArray}
    @boundscheck checkbounds(values, 1:getnbasefunctions(ip))
    @inbounds for i in 1:getnbasefunctions(ip)
```